base/base.bzl

```diff
@@ -126,17 +96,17 @@ def distro_components(distro_suffix):
     container_test(
         name = "base_release" + distro_suffix + "_test",
         configs = ["testdata/" + distro_suffix[1:] + ".yaml"],
-        image = ":base" + distro_suffix,
+        image = ":base_root_amd64" + distro_suffix,
     )
 
     container_test(
         name = "debug_release" + distro_suffix + "_test",
         configs = ["testdata/" + distro_suffix[1:] + ".yaml"],
-        image = ":debug" + distro_suffix,
+        image = ":debug_root_amd64" + distro_suffix,
     )
 
     container_test(
         name = "static_release" + distro_suffix + "_test",
         configs = ["testdata/" + distro_suffix[1:] + ".yaml"],
-        image = ":static" + distro_suffix,
+        image = ":static_root_amd64" + distro_suffix,
```

Comment:
do we need to add `container_test` blocks for all the `arm64` images?

Reply:
I didn't because then we'd need arm64 hardware to test them on presubmit, which starts to complicate things.

I've been cross-compiling this stuff and testing samples against a real arm64 cluster.  I hear that Bazel just started shipping arm64 binaries, so you might see if they have a solution for CI that this repo can piggyback on 🤔